app/build.gradle

```diff
@@ -110,8 +112,10 @@ dependencies {
     //waiting dots
     implementation "com.github.tajchert:WaitingDots:${rootConfiguration.waitingDotsVersion}"
 
+    //Youtube API
+    implementation files('libs/YouTubeAndroidPlayerApi.jar')
```

Comment:
F-Droid won't allow this

Reply:
@iamareebjamal making separate directories for FDroid and playstore and then adding the code